src/utils.js

```diff
@@ -210,12 +210,14 @@ var Utils = exports.Utils = {
    **/
   disablePrivacyBadgerForOrigin: function(origin){
     if(localStorage.disabledSites === undefined){
-      localStorage.disabledSites = JSON.stringify([origin]);
+      var disabledSites = new Object();
+      disabledSites[origin] = true;
+      localStorage.disabledSites = JSON.stringify(disabledSites);
```

Comment:
Each time somone disables privacy badger for a new site you would create an object {"newsite.com": true} and save that to localStorage.disabledSites, this would mean you could only have one disabled site at a time. This method should append to the disabled sites object not overwrite it. This could be solved by having an object in the global scope for utils.js that cached the contents of localStorage.disabledSites.